BackEnd/build.gradle

```diff
@@ -23,8 +24,18 @@ dependencies {
 	compileOnly 'org.projectlombok:lombok'
 	annotationProcessor 'org.projectlombok:lombok'
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
+	implementation 'org.flywaydb:flyway-core'
```

Comment:
move to `dependencies` in `buildScript` and use `classpath` instead of using `implementation`